index.html

```diff
@@ -0,0 +1,931 @@
+                        <h3 class="amt">
+                            $125M
+                            <br/>
+                            <span class="fundingType">SERIES C</span>
+                        </h3>
+
+                    </div>
+                </a>
+                <div class="aboutCompany">
+                    <a href="#" class="informationLineItem">
+                        <img src="Images/iconfinder_tag_1608866.png" class="icon" alt="Bookmark icon">
+                        <p class="textAlignCenter">Reviews and Recommendations</p>
+                    </a>
+                    <a href="#" class="informationLineItem">
+                        <img src="Images/iconfinder_briefcase_1608586.png" class="icon" alt="BriefCase icon">
+                        <p class="textAlignCenter">Summit Partners, +7</p>
+                    </a>
+                </div>
+                <div class="company">
+                    <a href="#" class="companyInfo">
+                        Utah’s Podium Raises $125M Series C, Rolls Out Free Version Of Its Product  <span class="greyScale">on CrunchBase News</span>
+                    </a>
+                </div>
+            </div>
+            <div class="company4">
+                <a href="#" class="noTextDecoration">
+                    <img src="https://img-cdn.tnwcdn.com/image?url=https%3A%2F%2Fpbs.twimg.com%2Fprofile_images%2F1077781538522185729%2FdDjJwzBa.jpg&amp;signature=2cf323c11b5ec80faad2524fd969d8b1" class="companyLogo" alt="Company Logo 4">
+                    <h4 class="fundedCompaniesFont">Swiggy</h4>
+                    <div class="fundingAmount">
+                        <h3 class="amt">
+                            $43M
+                            <br/>
+                            <span class="fundingType">SERIES H</span>
+                        </h3>
+
+                    </div>
+                </a>
+                <div class="aboutCompany">
+                    <a href="#" class="informationLineItem">
+                        <img src="Images/iconfinder_tag_1608866.png" class="icon" alt="Bookmark icon">
+                        <p class="textAlignCenter">Food Delivers, +3</p>
+                    </a>
+                    <a href="#" class="informationLineItem">
+                        <img src="Images/iconfinder_briefcase_1608586.png" class="icon" alt="BriefCase icon">
+                        <p class="textAlignCenter">Samsung Ventures, +4</p>
+                    </a>
+                </div>
+                <div class="company">
+                    <a href="#" class="companyInfo">
+                        Korean investors contribute $43m to India's Swiggy <span class="greyScale">on AVCJ</span>
+                    </a>
+                </div>
+            </div>
+        </section>
+        <section class="popularArticles">
+            <h2 class="switch">display none</h2>
+            <article class="populatArticle1">
+                <h2>
```

Comment:
The categories section has a different width than the first sections. This shouldn't happen (they have the same width).

Also, the space between rows is too large. Please adjust it.
Yours:
![Screenshot from 2020-04-13 13-29-59](https://user-images.githubusercontent.com/51926937/79148417-0b326680-7d8b-11ea-98d0-a08672fd122f.png)

Original:
![Screenshot from 2020-04-13 13-30-04](https://user-images.githubusercontent.com/51926937/79148437-14bbce80-7d8b-11ea-8077-5e7fec17d360.png)